8.x/theme-development/articles.md

```diff
@@ -1,3 +1,407 @@
 ## Articles
 
-[[toc]]
\ No newline at end of file
+[[toc]]
+
+On all sites, including store sites, there is a section where the content of the blog is introduced. A blog article is content written on a blog page. A blog page is an online journal or informational website usually belonging to a single individual or business that has its content displayed in reverse chronological order.
```

Comment:
On every website, including e-commerce ones, there is a section to manage the blog posts related to the business. So Larammerce also has this too.